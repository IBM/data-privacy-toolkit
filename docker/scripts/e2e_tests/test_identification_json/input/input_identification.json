[
  {
    "email": "foo@gmail.com",
    "name": "John",
    "city": "Paris",
    "continent": "Europe",
    "address": "P.O. BOX 1234",
    "atc": "A01AA01",
    "country": "Ireland",
    "gender": "Male",
    "marital": "Married",
    "county": "Bay County",
    "dependent": "Daughter",
    "numeric": 12345,
    "ip_address": "127.0.0.1",
    "race": "White",
    "street_type": "Avenue",
    "url": "http://www.google.com",
    "yob": 1985,
    "zip": "00647"
  }
]