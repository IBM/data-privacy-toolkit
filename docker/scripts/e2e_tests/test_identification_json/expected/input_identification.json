{"rawResults":{"/*/url":[{"typeName":"URL","count":1}],"/*/yob":[{"typeName":"NUMERIC","count":1}],"/*/street_type":[{"typeName":"UNKNOWN","count":1}],"/*/address":[{"typeName":"ADDRESS","count":1}],"/*/name":[{"typeName":"NAME","count":1}],"/*/country":[{"typeName":"COUNTRY","count":1}],"/*/atc":[{"typeName":"ATC","count":1}],"/*/numeric":[{"typeName":"NUMERIC","count":1}],"/*/city":[{"typeName":"CITY","count":1}],"/*/dependent":[{"typeName":"DEPENDENT","count":1}],"/*/marital":[{"typeName":"MARITAL_STATUS","count":1}],"/*/continent":[{"typeName":"CONTINENT","count":1}],"/*/zip":[{"typeName":"NUMERIC","count":1},{"typeName":"ZIPCODE","count":1}],"/*/ip_address":[{"typeName":"IP_ADDRESS","count":1}],"/*/county":[{"typeName":"UNKNOWN","count":1}],"/*/race":[{"typeName":"RACE","count":1}],"/*/gender":[{"typeName":"GENDER","count":1}],"/*/email":[{"typeName":"EMAIL","count":1}]},"bestTypes":{"/*/url":{"typeName":"URL","count":1},"/*/yob":{"typeName":"NUMERIC","count":1},"/*/street_type":{"typeName":"UNKNOWN","count":-1},"/*/address":{"typeName":"ADDRESS","count":1},"/*/name":{"typeName":"NAME","count":1},"/*/country":{"typeName":"COUNTRY","count":1},"/*/atc":{"typeName":"ATC","count":1},"/*/numeric":{"typeName":"NUMERIC","count":1},"/*/city":{"typeName":"CITY","count":1},"/*/dependent":{"typeName":"DEPENDENT","count":1},"/*/marital":{"typeName":"MARITAL_STATUS","count":1},"/*/continent":{"typeName":"CONTINENT","count":1},"/*/zip":{"typeName":"NUMERIC","count":1},"/*/ip_address":{"typeName":"IP_ADDRESS","count":1},"/*/county":{"typeName":"UNKNOWN","count":-1},"/*/race":{"typeName":"RACE","count":1},"/*/gender":{"typeName":"GENDER","count":1},"/*/email":{"typeName":"EMAIL","count":1}},"recordCount":1}